### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by tlipski**
new AngularJS selectors
webdriver/jvm timeout fixtures
make fill-form smarter
add url prefix support

### CODE REVIEW

File: src/io/aviso/taxi_toolkit/fixtures.clj [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- timeout-seconds - timeout in seconds after which the JVM should be stopped if the
+test is still running."
+  [timeout-seconds]
+  (fn [f]
+    (let [finished? (atom false)]
+      (doto (Thread. (fn []
+                       (Thread/sleep (* timeout-seconds 1000))
+                       (when-not @finished?
+                         (println "Timeout after " timeout-seconds "seconds.")
+                         (System/exit -100))))
```
Comment (kamituel) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: Negative return value? Not all shell's support that I think.

File: src/io/aviso/taxi_toolkit/fixtures.clj [hunk 2]
```diff
@@ -0,0 +1,74 @@
+(ns io.aviso.taxi-toolkit.fixtures
+  "General clojure.test purpose fixtures for running selenium/webdriver tests.
+  Please note that this namespace relies on webdriver-remote library."
+  (:require [clj-webdriver.taxi :as taxi]
+            [clj-webdriver.remote.server :as webdriver-remote])
+  (:import (java.util.concurrent TimeUnit)))
+
+(defn webdriver-remote-fixture
```
Comment (kamituel) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: Could this fn be somehow separated in two pieces, one general for setting up a driver, timeouts etc., and the other one that would build on top of that with remote session stuff?

File: src/io/aviso/taxi_toolkit/fixtures.clj [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                                                 host)
+                                            host))
+                                        "localhost")}
+                           browser-spec)]
+        (taxi/set-driver! a-driver)
+        (.setScriptTimeout (.. (:webdriver a-driver) manage timeouts) timeout-ms TimeUnit/MILLISECONDS) ;meh meh meh
+        (.pageLoadTimeout (.. (:webdriver a-driver) manage timeouts) timeout-ms TimeUnit/MILLISECONDS) ;meh meh meh
+        (taxi/implicit-wait 200)                            ;meh
+        (try
+          (taxi/window-resize {:width 1680 :height 1050})
```
Comment (kamituel) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: Could we make it a parameter?

File: src/io/aviso/taxi_toolkit/fixtures.clj (hunk 1 above)
Comment (kamituel) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: Yep, consider this:

``` bash
$ cat a.sh
exit -100
$ sh a.sh
$ echo $?
156
```

Comment (tlipski) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: Right, I will use 99 then

File: src/io/aviso/taxi_toolkit/fixtures.clj (hunk 3 above)
Comment (tlipski) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: Sure

File: src/io/aviso/taxi_toolkit/fixtures.clj (hunk 2 above)
Comment (tlipski) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: Yeah, I will split it even into 3 functions

File: src/io/aviso/taxi_toolkit/fixtures.clj (hunk 3 above)
Comment (tlipski) [inline src/io/aviso/taxi_toolkit/fixtures.clj]: but that makes a lot of options for a simple fixture, so I think we need to work on the structure of fixtures a little bit

**Comment by kamituel** [comment]
Thanks :)

**Pull request merged**